pkg/launch/launchable.go

```diff
@@ -17,9 +17,13 @@ type LaunchableID string
 
 func (l LaunchableID) String() string { return string(l) }
 
+type LaunchableVersionID string
```

Comment:
I agree that it's wordy, unfortunately `LaunchableVersion` already is a struct that includes not only the version identifier but also tags (e.g. os version)